.github/workflows/coding-standards.yml

```diff
@@ -7,8 +7,9 @@ on:
       - "master"
```

Comment:
I even belive, we should trigger CI for every PR as we might want to made PR into another PR

Reply:
done